src/main/java/baritone/process/MineProcess.java

```diff
@@ -421,9 +421,9 @@ private void addNearby() {
     }
 
     public static boolean plausibleToBreak(CalculationContext ctx, BlockPos pos) {
-        if (MovementHelper.getMiningDurationTicks(ctx, pos.getX(), pos.getY(), pos.getZ(), ctx.bsi.get0(pos), true) >= COST_INF) {
-            return false;
-        }
+//        if (MovementHelper.getMiningDurationTicks(ctx, pos.getX(), pos.getY(), pos.getZ(), ctx.bsi.get0(pos), true) >= COST_INF) {
```

Comment:
why is this gone lmao

Reply:
that whole functions only used for pathing goals and removing that allows pathing to "unbreakable" blocks ie. end portal frame. Also the avoid list is just copied from the hard coded values i removed.

Reply:
this is actually a hack, the intent of this check is actually to prevent setting the goal to a diamond ore that is adjacent to lava

Reply:
LOL based

Reply:
so is `getMiningDurationTicks` misnamed then?